docs/docs/pricing/index.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Pricing and API usage
+sidebar_position: 120
+sidebar_label: Pricing
+---
+
+When you use Flagsmith on Flagsmith-managed infrastructure (SaaS or private cloud), your pricing plan has a limit of API
+requests per month. This document describes how these limits work, and how to estimate how many API requests your
+applications will need.
+
+:::tip
+
+Self-hosted Flagsmith installations do not have API request limits.
+
+:::
+
+## How can I check my current usage?
+
+To check your API usage for the past 90 days:
+
+1. Log in to the Flagsmith dashboard as an organisation administrator.
+2. Click on your organisation name in the top left.
+3. Browse to the **Usage** tab.
+
+From this page, you can break down usage per project and per environment.
+
+The following screenshot shows an example usage graph from the Flagsmith dashboard:
+
+![img.png](usage-graph.png)
+
+## What counts as an API request?
+
+Any operation that calls Flagsmith's API to retrieve information about the current state of flags counts as 1 billable
```

Comment:
Hmm... this isn't completely true. We should also include something about identifying an identity and/or updating an identity's traits.

Reply:
Identifying an identity still returns the current state of flags - it's the current state of flags as seen by a specific user.

Updating an identity's traits also returns the current state of flags for the identity that is being updated.

I tried to make this as generic as possible while not overwhelming with technical details, considering the audience for this document includes people who have not used Flagsmith at all.

Reply:
I also tried to make this explicit in the example below:

> The device calls Flagsmith to update this user's traits and receives their latest flags in return (1 API request).

Reply:
I think we can solve any ambiguity here with a minor change to the wording: 

```suggestion
Any operation that calls Flagsmith's API to retrieve information about the current state of flags and/or persist an identity or their traits counts as 1 billable
```

Reply:
I think having this level of technical detail in the first sentence does more harm than good. We don't need to introduce the concepts of identity or trait at this point - I've specifically avoided using the word "identity" as a Flagsmith concept in this doc to make it accessible for a wider audience, especially less-technical decision makers who don't care about these details.

In a separate PR, I could add a dedicated section with details on what actual endpoints are billed, things like `/flags`, `/identities`, `/environment-document` etc to detail the actual specifics.

Reply:
Another way to word this that is technically more accurate and doesn't add complexity could be:

```suggestion
Any operation that calls Flagsmith's API and retrieves information about the current state of flags and/or persist an identity or their traits counts as 1 billable
```

Reply:
Yep, good with me, apart from one minor typo: `persist` -> `persists`